keda/post_install.md

```diff
@@ -0,0 +1,18 @@
+## KEDA
+
+KEDA is a Kubernetes-based Event Driven Autoscaler. With KEDA, you can drive the scaling of any container in Kubernetes based on the number of events needing to be processed.
+
+KEDA is a single-purpose and lightweight component that can be added into any Kubernetes cluster. KEDA works alongside standard Kubernetes components like the Horizontal Pod Autoscaler and can extend functionality without overwriting or duplication. With KEDA you can explicitly map the apps you want to use event-driven scale, with other apps continuing to function. This makes KEDA a flexible and safe option to run alongside any number of any other Kubernetes applications or frameworks.
+
+### Features
+
+- Event-driven 
+- Autoscaling Made Simple
```

Comment:
may add some examples aslo

Reply:
done, added an example for the Redis autoscaler. As there are many more, the link to the official doc should help too.